src/Cms/PageActions.php

```diff
@@ -247,6 +247,15 @@ protected function changeStatusToUnlisted()
         return $page;
     }
 
+    /**
+     * @param null $position
+     * @return self
+     */
+    public function changeSort($position = null)
```

Comment:
This should have a type hint:

```suggestion
    /**
     * @param int|null $position
     * @return self
     */
    public function changeSort(?int $position = null)
```